### PR TITLE
Amend doc 1.9 version

### PR DESCRIPTION
**Pull request by qiuming-best**
Signed-off-by: Ming <mqiu@vmware.com>

Thank you for contributing to Velero!

# Please add a summary of your change

# Does your change fix a particular issue?

Fixes #(issue)

# Please indicate you've done the following:

- [ ] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [ ] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 ## Older releases:
   * [CHANGELOG-1.8.md][18]
+  * [CHANGELOG-1.8.md][17]
```
Comment (blackpiglet) [inline CHANGELOG.md:6]: Looks like 1.7 changelog item missed, and 1.9 changelog should also be added.

Comment (qiuming-best) [reply CHANGELOG.md]: thanks!

1.9 changelog is in Line 2
1.7 changelog  item has been corrected

File: site/content/docs/v1.9/_index.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ If you encounter issues, review the [troubleshooting docs][30], [file an issue][
 
 ## Contributing
 
-If you are ready to jump in and test, add code, or help with documentation, follow the instructions on our [Start contributing](https://velero.io/docs/v1.9/start-contributing/) documentation for guidance on how to setup Velero for development.
+If you are ready to jump in and test, add code, or help with documentation, follow the instructions on our [Start contributing](https://velero.io/docs/v1.9.0/start-contributing/) documentation for guidance on how to setup Velero for development.
```
Comment (blackpiglet) [inline site/content/docs/v1.9/_index.md:36]: Not sure about that.
On velero.io, recent releases use v1.x as branch name.

Comment (qiuming-best) [reply site/content/docs/v1.9/_index.md:36]: yeah, I will modify it

**Review by ywk253100 — APPROVED** [review]

**Review by blackpiglet — APPROVED** [review]

**Pull request merged**